.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Why use `npx` instead of using `lint-staged` directly from development dependencies?

Reply:
Good question. I just followed this guide from the prettier docs https://prettier.io/docs/en/install.html#git-hooks